### PR TITLE
Simplify router configuration

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -957,6 +957,55 @@ services:
 
 </details>
 
+### How to run connect network of two projects
```
Comment (damienalexandre) [inline README.md:960]:
```suggestion
### How to run multiple projects simultaneously
```

File: README.md [hunk 2]
```diff
@@ -957,6 +957,55 @@ services:
...
+In the `foo` project, you'll need to declare the `bar_default` network in
+`docker-compose.yml`:
+
+```yaml
+networks:
+    bar_default:
+        external: true
+```
+
+Then, attach the router the bar network:
```
Comment (damienalexandre) [inline README.md:975]:
```suggestion
Then, attach it to the `foo` router:
```

**Review by damienalexandre — APPROVED** [review]

**Comment by lyrixx** [comment]
@damienalexandre Thanks for the review

**Review by pyrech — APPROVED** [review]
Thanks

**Pull request merged**